src/ts/components/subtitleselectbox.ts

```diff
@@ -34,7 +34,10 @@ export class SubtitleSelectBox extends SelectBox {
     };
 
     this.onItemSelected.subscribe((sender: SubtitleSelectBox, value: string) => {
+      // called twice? first time with null as String second time with NULL
```

Comment:
Where/how is it called twice?

Reply:
I forgot to dig into it.

Reply:
It's an issue within the `SelectBox` and its subclasses because the `onItemSelectedEvent` is triggered 1) when the `selectBoxValue` get changed or 2) the player events `ON_SUBTITLE_CHANGED` `ON_AUDIO_CHANGED`.

So far no problem but we set the values in the player (`setAudio` `setSubtitle`) when 1) is triggered and so 2) gets triggered immediately again. 
There is a mechanism in `ListSelector` to prevent these infinite event loop problem by checking the `selectedValue` but these value is null in the first time.

A fix would be to set the `selectedItem` in the `SelectBox` when the value of the DOM element changed.

_I hope this is understandable_

Reply:
As discussed offline, this is known behavior that isn't ideal, but not problematic either.